generated/nidcpower/README.rst

```diff
@@ -0,0 +1,201 @@
+Overall Status
```

Comment:
Was this change and other README.rst changes supposed to show up as moves from somewhere?

Reply:
Yes, generated already was where we were putting the generated files, but just not in a format that we could re-use. By `git mv`ing them we can keep the history of the files.